hooks/InnerNetClient.cpp

```diff
@@ -95,7 +95,7 @@ void dInnerNetClient_Update(InnerNetClient* __this, MethodInfo* method)
         if (State.originalName == "-") {
             app::GameData_PlayerOutfit* outfit = GetPlayerOutfit(GetPlayerData(*Game::pLocalPlayer));
             if (outfit != NULL)
-                State.originalName = convert_from_string(outfit->fields._playerName);
+                State.originalName = convert_from_string(outfit->fields.postCensorName);
```

Comment:
Should we use the postCensorName here or is there a better alternative?

Reply:
using postCensorName is the only way.(Decompiled from GameData_PlayerInfo$$get_PlayerName)